tests/SideBySide.New/AppConfig.cs

```diff
@@ -51,6 +51,12 @@ public static IConfiguration Config
 
 		public static bool SupportsJson => Config.GetValue<bool>("Data:SupportsJson");
 
+		public static string MySqlBulkLoaderCsvFile => Config.GetValue<string>("Data:MySqlBulkLoaderCsvFile");
+		public static string MySqlBulkLoaderLocalCsvFile => Config.GetValue<string>("Data:MySqlBulkLoaderLocalCsvFile");
+		public static string MySqlBulkLoaderTsvFile => Config.GetValue<string>("Data:MySqlBulkLoaderTsvFile");
+		public static string MySqlBulkLoaderLocalTsvFile => Config.GetValue<string>("Data:MySqlBulkLoaderLocalTsvFile");
+		public static bool MySqlBulkLoaderRemoveTables => Config.GetValue<bool>("Data:MySqlBulkLoaderRemoveTables");
```

Comment:
Is this just for local testing (i.e., to manually inspect the tables after the tests run)?